bahmni-emr-api/src/main/java/org/openmrs/module/bahmniemrapi/laborder/mapper/LabOrderResultMapper.java

```diff
@@ -88,9 +92,28 @@ private Obs newResultObs(Order testOrder, Date obsDate, Concept concept, LabOrde
             return null;
         }
         obs.setValueAsString(labOrderResult.getResult());
+        checkResultRangesForAbsolutes(obs, accessionUuid);
         return obs;
     }
 
+    /**
+     * This method just logs error if the results are out of absolute ranges. This will be errored out (ValidationException)
+     * by openmrs by the {@link org.openmrs.validator.ObsValidator} during save
+     */
+    private void checkResultRangesForAbsolutes(Obs obs, String accessionUuid) {
+        if (obs.getConcept().isNumeric()) {
```

Comment:
Minor: Reverse as Guard clause.
if(!Numeric) return;

Reply:
sure. It would still eval one if statement though (and it does not have any other if). But the subsequent block can lose an if statement. Will do